app/models/manageiq/providers/amazon/cloud_manager/ssa_agent_manager.rb

```diff
@@ -211,38 +211,24 @@ def create_security_group(group_name = MIQ_SSA)
         vpc_id: ec2.client.describe_vpcs.vpcs[0].vpc_id
       })
 
-      security_group.authorize_ingress({
-        ip_permissions: [{
-          ip_protocol: 'tcp',
-          from_port: 22,
-          to_port: 22,
-          ip_ranges: [{
-            cidr_ip: '0.0.0.0/0'
-          }]}]
-      })
+      sg_authorize_ingress(security_group, 'tcp', 22, 22)
+      sg_authorize_ingress(security_group, 'tcp', 80, 80)
+      sg_authorize_ingress(security_group, 'tcp', 443, 443)
 
-      security_group.authorize_ingress({
-        ip_permissions: [{
-          ip_protocol: 'tcp',
-          from_port: 80,
-          to_port: 80,
-          ip_ranges: [{
-            cidr_ip: '0.0.0.0/0'
-          }]}]
-      })
+      security_group.group_id
+    end
+  end
 
-      security_group.authorize_ingress({
+  def sg_authorize_ingress(sg, protocal, from_port, to_port)
```

Comment:
You misspelled the second argument as "protocal" instead of "protocol".  you spelled it correctly on line 225 so this needs to be changed.